sentinelhub/data_utils.py

```diff
@@ -83,6 +83,16 @@ def _extract_response_data(response_data: List[JsonDict], exclude_stats: List[st
     return df_entries
 
 
+def _is_batch_stat(result_data: JsonDict) -> bool:
+    """Check statistical request type
+
+    :param request_data: An input representation of (Batch) Statistical API result
+        returned from `DataRequest.get_data()`.
+    :return: bool
+    """
```

Comment:
```suggestion
    """Identifies whether the resulting data belongs to a batch statistical request or not."""
```

I suppose this is the idea of this function?

Reply:
i see you changed the first line, but i would remove the docstring about parameters as well
the typess say all that one needs, especially since the `request_data` description seems odd